src/data/game.py

```diff
@@ -0,0 +1,47 @@
+track_slugs = [
```

Comment:
Python convention tends to prefer to have global constants be capitalized. Worth looking into it and changing these to all caps.

Reply:
Yeah well.. DEAL WITH IT

jk jk

Reply:
Fixed!